code/src/index.css

```diff
@@ -1,13 +1,154 @@
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #181818;
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+a:link {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+a:active {
+  text-decoration: none;
+}
+
+hr {
+  border: 1px solid #212121;
+}
+
+h1 {
+  margin: 0px;
+  font-size: 24px;
+}
+
+.main-container {
+  padding: 20px;
+}
+
+.albums-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-column-gap: 20px;
+  height: max-content;
+  justify-items: center;
+  align-items: stretch;
+}
+
+.album-box {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding-bottom: 40px;
+}
+
+.album-cover {
+  filter: brightness(100%);
+  opacity: 1;
```

Comment:
You could probably remove this opacity: 1 as there is nothing else in the css changing the opacity.